dropshot/Cargo.toml

```diff
@@ -16,6 +16,7 @@ async-trait = "0.1.56"
 base64 = "0.13.0"
 bytes = "1"
 futures = "0.3.21"
+headers = "^0.3.7"
```

Comment:
Do without the carets just to stay consistent. They mean the same anyways

Reply:
oh will do